lib/base/heimbase-svc.h

```diff
@@ -68,6 +68,7 @@
     char *cname;                                                \
     char *sname;                                                \
     const char *e_text;                                         \
+    heim_octet_string *e_data;                                  \
```

Comment:
I guess I thought it might be useful to be able to force the e-data field to be empty (by calling set_e_data() with zero-length data), but the way you suggest is probably better.